tests/test.c

```diff
@@ -2258,7 +2259,6 @@ test_consume_msgs_easy (const char *group_id, const char *topic,
         test_msgver_clear(&mv);
 }
 
-
```

Comment:
whitespace. Please scan through your diff on github, I find it really helps seeing the same code in a different setting.